documentation-website/Writerside/topics/Working-with-Database.md

```diff
@@ -158,7 +175,7 @@ Database.connect(
 Add the dependency:
 
 ```kotlin
-implementation("org.xerial:sqlite-jdbc:3.30.1")  
+implementation("org.xerial:sqlite-jdbc:3.44.1.0")
```

Comment:
About the link to FAQ below: I'm not sure what exactly that link was supposed to originally go to? But there's nothing currently on that page relating to SQLite, so maybe this might be more useful: https://www.sqlite.org/isolation.html